src/main/scala/io/iohk/ethereum/network/handshaker/EtcHelloExchangeState.scala

```diff
@@ -26,19 +26,15 @@ case class EtcHelloExchangeState(handshakerConfiguration: EtcHandshakerConfigura
 
   override def applyResponseMessage: PartialFunction[Message, HandshakerState[PeerInfo]] = { case hello: Hello =>
     log.debug("Protocol handshake finished with peer ({})", hello)
-    if (
-      handshakerConfiguration.protocolVersion == ProtocolVersions.PV64 && hello.capabilities.contains(Etc64Capability)
-    )
-      EtcNodeStatus64ExchangeState(handshakerConfiguration)
-    else if (hello.capabilities.contains(Eth63Capability))
-      EtcNodeStatus63ExchangeState(handshakerConfiguration)
-    else {
-      log.debug(
-        s"Connected peer does not support eth ${ProtocolVersions.PV63.toByte} / ${ProtocolVersions.PV64.toByte} protocol. Disconnecting."
-      )
-      DisconnectedState(Disconnect.Reasons.IncompatibleP2pProtocolVersion)
+    capability match {
+      case ProtocolVersions.ETC64 => EtcNodeStatus64ExchangeState(handshakerConfiguration)
+      case ProtocolVersions.ETH63 => EtcNodeStatus63ExchangeState(handshakerConfiguration)
+      case _ =>
+        log.debug(
+          s"Connected peer does not support ${ProtocolVersions.ETH63} / ${ProtocolVersions.ETC64} protocol. Disconnecting."
```

Comment:
shall we align the log msg format?
```suggestion
          "Connected peer does not support {} / {} protocol. Disconnecting.", ProtocolVersions.ETH63, ProtocolVersions.ETC64
```